dae/dae/gene/gene_sets_db.py

```diff
@@ -17,30 +17,31 @@ def __init__(self, gene_sets_collection_id, config):
         self.gene_sets_descriptions = None
         self.gene_sets_collections = None
 
-    def _load(self):
-        try:
-            gene_sets_collection = \
-                GeneInfoConfigParser.getGeneTerms(self.config, self.gsc_id)
-        except Exception:
-            traceback.print_exc()
-            gene_sets_collection = loadGeneTerm(self.gsc_id)
+    @staticmethod
+    def _load_collection(config, gene_sets_collection):
```

Comment:
this should be renamed and reorganized to match what method really do...

Reply:
probably the constructor should accept GeneInfo instance and use that instance when needed...